### PR TITLE
[Logs]: Add change notice for Cloudflare Logpush User-Agent

### PR DESCRIPTION
**Pull request by cmackenzie1**

### CODE REVIEW

**Comment by kodster28** [comment]
Howdy @cmackenzie1, this seems like something that also might warrant being included in the changelog.

Onboarding instructions in our [style guide](https://developers.cloudflare.com/style-guide/documentation-content-strategy/content-types/changelog/#structure-single-page).... but also happy to jump into the PR if it'd be helpful.

**Review by kodster28 — APPROVED** [review]

**Review by ngayerie — APPROVED** [review]

**Comment by cmackenzie1** [comment]
Going to reschedule this to something ~60 days out and get an email sent out to Logpush customers.

File: data/changelogs/logs.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+link: "/logs/change-notices/"
+productName: Logs
```
Comment (kodster28) [inline data/changelogs/logs.yaml:3]:
```suggestion
productName: Logs
productLink: "/logs/"
```

Comment (kodster28) [reply data/changelogs/logs.yaml:3]: Just pushed up a change in the template around this.

**Comment by angelampcosta** [comment]
@cmackenzie1 any updates on this PR? It's showing some conflicts because of the migration.

**Comment by cmackenzie1** [comment]
@angelampcosta Closing the PR for now. Will reopen once we are closer to making this change.

**Pull request closed without merging**